### PR TITLE
[feat] Add commands of math formulas

### PR DESCRIPTION
**Pull request by MakarenkoAI**
./bin/scn-tex2scs -s Contents -t example

### CODE REVIEW

File: src/grammar/SCnTexLexer.cpp [hunk 1]
```diff
@@ -1,5 +1,5 @@
 
-// Generated from ./src/grammar/SCnTex.g4 by ANTLR 4.7.1
+// Generated from /home/anna/projects/tex2scs-translator/src/grammar/SCnTex.g4 by ANTLR 4.7.2
```
Comment (MksmOrlov) [inline src/grammar/SCnTexLexer.cpp]: absolute path is not a good idea, fix it everywhere in PR

**Review by MksmOrlov — CHANGES_REQUESTED** [review]
update tests/documentation for math please

File: CMakeLists.txt [hunk 2]
```diff
@@ -1,7 +1,9 @@
-cmake_minimum_required(VERSION 2.8)
+cmake_minimum_required(VERSION 2.8.12)
```
Comment (NikitaZotov) [inline CMakeLists.txt]:
```suggestion
cmake_minimum_required(VERSION 3.11)
```

File: CMakeLists.txt [hunk 3]
```diff
@@ -1,7 +1,9 @@
...
 project(sc-scn-tex2scs-translator)
 cmake_policy(SET CMP0048 NEW)
 set(TRANSLATOR_ROOT ${CMAKE_CURRENT_LIST_DIR})
+set(CMAKE_BUILD_TYPE Debug)
```
Comment (NikitaZotov) [inline CMakeLists.txt]:
```suggestion
```

File: README.md [hunk 4]
```diff
@@ -41,3 +41,6 @@ Tex2SCs-translator is a tool for translating SCn-represented *.tex files into SC
 
 Contributions, bug reports and feature requests are welcome!
 Feel free to check our [issues page](https://github.com/ostis-ai/tex2scs-translator/issues) and file a new issue (or comment in existing ones).
+
+\scnlegtcurlbrace element\scnrightcurlbrace 
+With space!
```
Comment (NikitaZotov) [inline README.md]:
```suggestion
```

File: src/translator/commands/math/sc_scn_tex_operation_command.h [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+{
+protected:
+  uint8_t const COMMAND_POS = 0;
+  uint8_t const FIRST_PARAM_POS = 1;
+  uint8_t const SECOND_PARAM_POS = 2;
+
+  uint8_t const A_ATTR_POS = 0;
+  uint8_t const B_ATTR_POS = 1;
+  uint8_t const C_ATTR_POS = 2;
+  uint8_t const D_ATTR_POS = 3;
```
Comment (NikitaZotov) [inline src/translator/commands/math/sc_scn_tex_operation_command.h]: Please, provide more meaningful names for this values

File: src/translator/commands/math/sc_scn_tex_math_command.cpp [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+      return "";
+    else 
+      return symbolItem->second.at(0);
+  }
+
+  return SCsStream()
+    .Row([&](SCsStream & stream) {
+      auto const attrs = item->second;
+
+      auto const openBracketOfFirstArg = [&]() -> std::string {
```
Comment (NikitaZotov) [inline src/translator/commands/math/sc_scn_tex_math_command.cpp]:
```suggestion
      auto const OpenBracketOfFirstArg = [&]() -> std::string {
```

File: src/translator/commands/math/sc_scn_tex_math_command.cpp [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+        return attrs[C_ATTR_POS];
+      };
+
+      auto const closedBracketOfSecondArg = [&]() -> std::string {
+        return attrs[D_ATTR_POS];
+      };
+
+      if (attrs.size() == COMMAND_SIZE_ONE_ARG)
+      {
+        std::string const & first_argument = params.at(FIRST_PARAM_POS);
```
Comment (NikitaZotov) [inline src/translator/commands/math/sc_scn_tex_math_command.cpp]:
```suggestion
        std::string const & firstArgument = params.at(FIRST_PARAM_POS);
```

Comment (NikitaZotov) [reply src/translator/commands/math/sc_scn_tex_math_command.cpp]: Review all your code style. It should be a code style of the main code

**Review by NikitaZotov — CHANGES_REQUESTED** [review]

**Review by MksmOrlov — APPROVED** [review]

**Review by NikitaZotov — APPROVED** [review]

**Pull request merged**